lib/ClangImporter/ImportDecl.cpp

```diff
@@ -9761,6 +9799,9 @@ ClangImporter::Implementation::loadAllMembers(Decl *D, uint64_t extra) {
 
   // If not, we're importing globals-as-members into an extension.
   if (objcContainer) {
+    // TODO: Similar to with structs and enums below, it looks like getMembers
```

Comment:
@zoecarver could you provide some more details on what the "hack" below accomplishes? From what I can tell it runs `FirstDeclAndLazyMembers.setInt(true);` which helps keep future lookups lazy, but I could use some clarification.